### DISCUSSION TITLE
Images/Css not loading when Basic Auth is activated

### DISCUSSION THREAD

**Discussion by DGINXREAL** [mpdf/mpdf#2018] [General]
Hello,

I have currently a customer he uses basic auth on his website because it's still under development. Now i see a trouble. Images and CSS are not loading from server because of basic auth. If basic auth is disabled it works.

Is that possible to provide basic auth credentials when mdpf try to download images/Css?

**Comment by finwe** [comment]
You should be able to provide the credentials [in path directly](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#access_using_credentials_in_the_url) as `http(s)://username:password@path_to_image`

Or you can define and provide [your own Http client implementation](https://mpdf.github.io/configuration/configuration-v7-x.html#http-client) to fetch resources with credentials.
